### PR TITLE
PS-8084 Create 5.7.37-40 release notes and release note tasks 	

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   source/conf.py
new file:   source/release-notes/Percona-Server-5.7.37-40.rst
modified:   source/release-notes/release-notes_index.rst

### CODE REVIEW

**Review by george-lorch — APPROVED** [review]
+1

File: doc/source/release-notes/Percona-Server-5.7.37-40.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+* The performance on debug builds has been improved by optimazing `buf_validate()` function in the *InnoDB* sources.
+* Fix for when a query using an index that differs from the primary key of partitioned table results in excessive CPU load.
+
+Find the complete list of bug fixes and changes in `MySQL 5.7.37 Release Notes <https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-37.html>`__.
+
+Improvements
+=================================================
+
+* :jirabug:`PS-7792`: Fix for when replication fails if `Master_User` that is not stored in `slave_master_info` on the replica is blank.
```
Comment (patrickbirch) [inline doc/source/release-notes/Percona-Server-5.7.37-40.rst]: Remove "Fix for" - make present tense

File: doc/source/release-notes/Percona-Server-5.7.37-40.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Bugs Fixed
+=================================================
+
+* :jirabug:`PS-7929`: Fix for when the row locks are duplicated when inserting an existing row into a table within the same transaction.
+* :jirabug:`PS-8007`: *Percona Server for MySQL* can fail to start if the server starts before the network mounts the datadir or a local mount of the datadir.
+* :jirabug:`PS-7856`: The server exits during the partition tables update.
```
Comment (patrickbirch) [inline doc/source/release-notes/Percona-Server-5.7.37-40.rst]: Make past tense

File: doc/source/release-notes/Percona-Server-5.7.37-40.rst [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+* :jirabug:`PS-7890`: The installation of RocksDB engine plugin fails if the server is started with `--loose-rocksdb_persistent_cache_size_mb`.
```
Comment (patrickbirch) [inline doc/source/release-notes/Percona-Server-5.7.37-40.rst]: past tense

Comment (alina-derkach-oaza) [reply doc/source/release-notes/Percona-Server-5.7.37-40.rst]: Updated

File: doc/source/release-notes/Percona-Server-5.7.37-40.rst (hunk 2 above)
Comment (alina-derkach-oaza) [reply doc/source/release-notes/Percona-Server-5.7.37-40.rst]: Done

File: doc/source/release-notes/Percona-Server-5.7.37-40.rst (hunk 1 above)
Comment (alina-derkach-oaza) [reply doc/source/release-notes/Percona-Server-5.7.37-40.rst]: Fixed

File: doc/source/release-notes/Percona-Server-5.7.37-40.rst [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+* The GnuPG build key has been updated. The GnuPG build key is used to sign MySQL downloadable packages. The systems configured to use repo.mysql.com may report a signature verification error when upgrading using *apt* or *yum* to MySQL 5.7.37 and higher or to MySQL 8.0.28 and higher. To resolve this issue, use one of the following methods:
+
+  * Reinstall the MySQL APT or YUM repository setup package from https://dev.mysql.com/downloads/.
+
+  * Download the MySQL GnuPG public key and add it to your system GPG keyring.
+
+  * Find MySQL APT repository instructions in `Appendix A: Adding and Configuring the MySQL APT Repository Manually <https://dev.mysql.com/doc/mysql-apt-repo-quick-guide/en/#repo-qg-apt-repo-manual-setup>`__.
+
+  * Find MySQL YUM repository instructions in `Upgrading MySQL with the MySQL Yum Repository <https://dev.mysql.com/doc/mysql-yum-repo-quick-guide/en/#repo-qg-yum-upgrading>`__.
```
Comment (hrvojem) [inline doc/source/release-notes/Percona-Server-5.7.37-40.rst]: this part can be removed as this concerns Oracle's repos and packages.

**Review by hrvojem — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/Percona-Server-5.7.37-40.rst (hunk 4 above)
Comment (alina-derkach-oaza) [reply doc/source/release-notes/Percona-Server-5.7.37-40.rst]: Removed

**Review by hrvojem — APPROVED** [review]

**Pull request merged**